ChatSX/constants/Numbers.ts

```diff
@@ -0,0 +1,10 @@
+export const TWO = 2;
+export const THREE = 3;
+export const FOUR = 4;
+export const SIX = 6;
+export const EIGHT = 8;
+export const NINE = 9;
+export const FIFTEEN = 15;
+export const TWENTY_ONE = 21;
+export const TWENTY_SEVEN = 27;
+export const TWENTY_EIGHT = 28;
```

Comment:
what are these enum? why do we enums for counting, enums are used to specific what value it is representing, for example `id==3` represent menu it for `Share`, so enum name should `ITEM_SHARE=3` and not `THREE=3`.